ksqldb-engine/src/main/java/io/confluent/ksql/query/QueryRegistryImpl.java

```diff
@@ -81,23 +82,26 @@ public QueryRegistryImpl(final Collection<QueryEventListener> eventListeners) {
   private QueryRegistryImpl(final QueryRegistryImpl original) {
     executorFactory = original.executorFactory;
     persistentQueries = new ConcurrentHashMap<>();
-    allLiveQueries = ConcurrentHashMap.newKeySet();
+    allLiveQueries = new ConcurrentHashMap<>();
     createAsQueries = new ConcurrentHashMap<>();
     insertQueries = new ConcurrentHashMap<>();
-    original.allLiveQueries.forEach(query -> {
-      if (query instanceof PersistentQueryMetadata) {
-        final PersistentQueryMetadata sandboxed = SandboxedPersistentQueryMetadataImpl.of(
-            (PersistentQueryMetadataImpl) query,
-            new ListenerImpl()
-        );
-        persistentQueries.put(sandboxed.getQueryId(), sandboxed);
-        allLiveQueries.add(sandboxed);
-      } else {
-        final TransientQueryMetadata sandboxed = SandboxedTransientQueryMetadata.of(
-            (TransientQueryMetadata) query,
-            new ListenerImpl()
-        );
-        allLiveQueries.add(sandboxed);
+    original.allLiveQueries.forEach(new BiConsumer<QueryId, QueryMetadata>() {
```

Comment:
You can just simplify this to a lambda `(queryId, queryMetadata) -> {` to remove a bunch of the below boilerplate.